Source/WebKit/UIProcess/API/APIPageConfiguration.h

```diff
@@ -509,6 +509,7 @@ class PageConfiguration : public ObjectImpl<Object::Type::PageConfiguration> {
         HashMap<WTF::String, Ref<WebKit::WebURLSchemeHandler>> urlSchemeHandlers;
         Vector<WTF::String> corsDisablingPatterns;
         HashSet<WTF::String> maskedURLSchemes;
+        bool maskedURLSchemesWasSet { false };
```

Comment:
I thought about having a nit that this should be `maskedURLSchemesWereSet` for better subject/verb agreement, but if you think of `maskedURLSchemes` as a singular concept, then `was` is more correct.